src/repos/PublicationRepo.ts

```diff
@@ -0,0 +1,47 @@
+import { ObjectId } from 'mongodb';
+import { Publication, PublicationModel } from '../entities/Publication';
+import publicationsJSON from '../../publications.json';
+
+/**
+ * Reads publication info from static JSON and stores info in database.
+ * @function
+ * @returns {Publication []} An array of Publication objects.
+ */
+const addPublicationsToDB = async (): Promise<void> => {
+  const publicationsDB = publicationsJSON.publications;
+
+  let publications = [];
+  for (const publication of publicationsDB) {
+    const { bio, rssURL, imageURL, name, websiteURL, rssName } = publication;
+    publications.push(Object.assign(new Publication(), {
+      _id: new ObjectId().toString(),
+      bio,
+      rssURL,
+      imageURL,
+      name,
+      websiteURL,
+      rssName,
+      shoutouts: 0
```

Comment:
if default shoutouts is zero, we should be able to set that in the entity definition. Better to define this intended behavior at the model level vs de-coupling it in the repo level.